plugins/single_plugins/resize.cpp

```diff
@@ -93,14 +95,29 @@ class wayfire_resize : public wf::per_output_plugin_instance_t, public wf::point
         output->connect(&on_view_disappeared);
     }
 
+    bool activate(bool should_preserve_aspect)
```

Comment:
There is one more way to activate the plugin, see `on_resize_request` defined in the beginning of the class.

Reply:
I've set `preserve_aspect` to `false` to keep the behavior.